javascript/sdk/lib/controllers/detect.ts

```diff
@@ -168,6 +168,7 @@ export default class DetectController {
     if (query.result_id) searchParams.set("result_id", query.result_id);
```

Comment:
activity filters in the BE are:
- start
- finish
- tests
- dos
- endpoints
- statuses
- control

Reply:
no impersonate?

Reply:
It should be there since we are using that for social activity for metrics, tests, and advisories views

Reply:
oh yeah, that too. wasn't really considering it a filter, but suppose it is 👍